src/app/services/candidate-context.service.ts

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+
+import { candidateRequestData } from '../models/candidate-request-data.model';
+import { ApiService } from './api.service';
+import { IdName } from '../models/id-name.model';
+import { Sandbox } from '../models/sandbox.model';
+import { CandidateFormModel } from '../models/candidateForm.model';
+
+import { ToastrService } from 'ngx-toastr';
+import { TranslateService } from '@ngx-translate/core';
+
+@Injectable({ providedIn: 'root' })
+export class CandidateContextService extends ApiService<candidateRequestData> {
+  constructor(
+    private http: HttpClient,
+    private toastr: ToastrService,
+    private translateService: TranslateService,
+  ) {
+    super(http, candidateRequestData, '');
+    this.translateService.onLangChange.subscribe(() => {
+      this.translateLabels();
+    });
+    this.translateLabels();
+  }
+
+  title = '';
+  text = '';
+
+  englishLevels: IdName[] = [];
+  skills: IdName[] = [];
+  availability: IdName[] = [];
+  sandboxes: Sandbox[] = [];
+
+  getData(dataArray: any[], currentUrl: string): any[] {
```

Comment:
please, change any[] type to something else. According to code below it should be candidateRequestData[]